pallets/token-wrapper/src/traits.rs

```diff
@@ -1,6 +1,9 @@
 use frame_support::dispatch;
 
 pub trait TokenWrapperInterface<AccountId, AssetId, Balance> {
+	fn set_wrapping_fee(
```

Comment:
shouldn't need to use line breaks here, it looks like it can be on one line

Reply:
fixed